docs/react-wiki-archive/Contributing/New-Components.md

```diff
@@ -0,0 +1,87 @@
+# Contributing a new component to Fluent UI React
+
+Our components are a collection of small, independent, general-purpose UI components intended to be reused throughout an application.
```

Comment:
This file contains a lot of process information that could be relevant. It was last updated in April 2022.

Reply:
PR Buddy == Tudor, Be consistent with Convergence Epic template

Reply:
pr created: #31611